base/initdefs.jl

```diff
@@ -238,7 +238,8 @@ function init_active_project()
     set_active_project(
         project === nothing ? nothing :
         project == "" ? nothing :
-        startswith(project, "@") ? load_path_expand(project) : abspath(expanduser(project))
+        startswith(project, "@") ? load_path_expand(project) :
+        abspath(expanduser(project))
```

Comment:
Is this just a formatting change? From a formatting perspective, I think indentation of line 242 would be useful.

From a soruce control perspective, I would prefer no change since this does not seem necessary for the feature being added
.
```suggestion
        startswith(project, "@") ? load_path_expand(project) :
            abspath(expanduser(project))
```

Reply:
Indenting just the line 242 wouldn't do, considering previous 2 lines are also the else-options for ?:
But I agree, this change isn't necessary. I've reverted that [here](https://github.com/JuliaLang/julia/compare/75a85a627a09569f1cec50a93e7191707321d08b..b203b09a46c7afebf23e8fdfa5793b2e081b71af).